### DISCUSSION TITLE
How many Ampere are needed for which track sizes?

### DISCUSSION THREAD

**Discussion by probonopd** [gab-k/RP2040-Decoder#8] [Q&A]
I noticed that commercial decoders ar priced (very) differently for H0 vs. Spur 1, with decoders for the latter usually offering more Ampere (e.g., 3 to 5 Ampere) for (much) more money.

So, how many Ampere do we need for the different track sizes? And how to find out (measure) this easily?

**Comment by gab-k** [comment]
> So, how many Ampere do we need for the different track sizes? And how to find out (measure) this easily?

It's hard to say, it depends on the make, model and the particular engine you're dealing with (also the speed and how fast you are accelerating / duty cycle). What I can say, however, is that you should take the 2.8 A stated in the BDR6133 datasheet with a grain of salt, since an R_DSON of ~0.5 ohms and a current of 2.8 amperes result in about 4 W of power loss (no switching losses included). The Chinese data sheet is rather optimistic. However, there is a thermal shutdown so you could just take your chances. There are a number of ways to measure current, but I'm reluctant to go into detail as I'm sure there are plenty of articles on the internet on the subject such as this one: http://www.circuitous.ca/DCCvolts.html
